README.md

```diff
@@ -5,7 +5,7 @@
 
 ###GATK 4 (codename Hellbender)
 
-The public parts of the next generation of GATK/Picard methods engine and tools.
+This repository contains the public parts of the next generation of GATK/Picard methods engine and tools). See also [gatk-protected](https://github.com/broadinstitute/gatk-protected) for the parts of the toolkit that are licence-protected.
```

Comment:
I might rephrase this as something like "protected by a license that restricts their use to non-commercial applications". 

We should probably add a note at some point explaining the two-repo structure a bit more clearly. Feel free to assign an issue to me to do that.